ns/json-ld.ttl

```diff
@@ -10,41 +10,55 @@
 jsonld: a owl:Ontology;
   dc:title "The JSON-LD Vocabulary"@en;
   dc:description """This is a vocabulary document and is used to achieve certain features of the JSON-LD language."""@en;
-  dc:date "2019-01-04"^^xsd:date;
-  owl:versionInfo <https://github.com/w3c/json-ld-wg/commit/f8ca95bda51a6c7bc8ccd7f63b150d602010c478>;
+  dc:date "2020-01-03"^^xsd:date;
+  owl:versionInfo <https://github.com/w3c/json-ld-wg/commit/c5b6145b2182aab77b17db65d8367118c1a19b19>;
```

Comment:
This auto generated hash here and in other files is a bit odd as it can only refer to the previous commit.  Yet this commit has relevant changes.  Is that good enough for whatever purpose this is for?

Reply:
It makes something unique, but you're right that it is off-by-one. No real way to know the commit before it's generated.

As this is versionInfo, it doesn't really need to correlate 1-1. We could consider creating a tag mechanism and reference a tag, which can then be associated with a commit, but I don't think it's important enough to require such a manual step.